src/openrct2/drawing/Line.cpp

```diff
@@ -141,14 +140,14 @@ void gfx_draw_line_software(rct_drawpixelinfo* dpi, const ScreenLine& line, int3
     {
         // Vertical lines are drawn 1 pixel at a time
         if (steep)
-            gfx_draw_line_on_buffer(dpi, colour, x, y, 1);
+            gfx_draw_line_on_buffer(dpi, colour, { y, x }, 1);
```

Comment:
Very weird that X and y were swapped, maybe needs further investigation in the future

Reply:
I'm not too sure why it was swapped in the function definition...

Reply:
Yeah, but this was the only place calling it swapped, which looks like an error, maybe just an oversight.

Reply:
Can you try checking what the function is doing and the impact of swapping it to the supposedly correct state, please?

Reply:
y,x
![image](https://user-images.githubusercontent.com/12629817/117551804-071c5b00-b016-11eb-91f7-7630d72904bf.png)

x,y
![image](https://user-images.githubusercontent.com/12629817/117551918-890c8400-b016-11eb-80e8-1cdc57fde126.png)

Reply:
It's only used for the plots I believe.